Retrovizor.Data/Entities/RetrovizorContext.cs

```diff
@@ -86,16 +98,103 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .HasOne(r => r.Student)
                 .WithMany(r => r.Reviews)
                 .HasForeignKey(r => r.StudentId);
-            
+
             // overriding default delete behaviour
             var cascadeFKs = modelBuilder.Model.GetEntityTypes()
                 .SelectMany(t => t.GetForeignKeys())
                 .Where(fk => !fk.IsOwnership && fk.DeleteBehavior == DeleteBehavior.Cascade);
 
-            foreach(var fk in cascadeFKs)
+            foreach (var fk in cascadeFKs)
                 fk.DeleteBehavior = DeleteBehavior.ClientSetNull;
 
             base.OnModelCreating(modelBuilder);
+
+            // test seed data
```

Comment:
Makni ove seedove zbog seed branch konflikata.

Reply:
Zaboravija na to, ostalo mi od testiranja, micen.

Reply:
Zaboravija na to, ostalo mi od testiranja, micen.